src/array_api_stubs/_draft/elementwise_functions.py

```diff
@@ -852,15 +883,15 @@ def conj(x: array, /) -> array:
     """
 
 
-def copysign(x1: array, x2: array, /) -> array:
+def copysign(x1: Union[array, float], x2: Union[array, float], /) -> array:
```

Comment:
(obviously this is an exception to the "allow int in float functions" rule)

Reply:
I went ahead and added explicit `int` support as we should be consistent across all multi-argument element-wise functions.